Add --log-exporter option to the linux installer script (#1500)
splunk/public-o11y-docs: gdi/opentelemetry/collector-linux/install-linux.rst

   * - ``--metrics-exporter <exporters>``
     - Comma-separated list of exporters for collected metrics by all activated SDKs, for example ``otlp,prometheus``. Set the value to ``none`` to disable collection and export of metrics. The value will be set to the ``OTEL_METRICS_EXPORTER`` environment variable. The default behavior is to defer to the default ``OTEL_METRICS_EXPORTER`` value for each activated SDK.
     -
   * - ``--logs-exporter <exporter>``

>      - Set the exporter for collected logs by all activated SDKs, for example ``otlp``. Set the value to ``none`` to disable collection and export of logs. The value will be set to the ``OTEL_LOGS_EXPORTER`` environment variable. The default behavior is to defer to the default ``OTEL_LOGS_EXPORTER`` value for each activated SDK.

Reviewer: ```suggestion
     - Set the exporter for collected logs by all activated SDKs, for example ``otlp``. Set the value to ``none`` to deactivate collection and export of logs. The ``OTEL_LOGS_EXPORTER`` environment variable contains this value. By default, each activated SDK uses the default ``OTEL_LOGS_EXPORTER`` value.
```
